miden-lib/asm/miden/kernels/tx/tx.masm

```diff
@@ -59,10 +59,10 @@ const.ERR_NON_FUNGIBLE_ASSET_ALREADY_EXISTS=0x00020051
 # =================================================================================================
 
 # Event emitted when a new note is created.
-const.NEW_NOTE_EVENT=131077
+const.NEW_NOTE_EVENT=131082
```

Comment:
Should we rename this to `BEFORE_NOTE_CREATED_EVENT` and also add `AFTER_NOTE_CREATED_EVENT` event?